modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java

```diff
@@ -632,7 +632,9 @@ public AndesSubscription getSubscriptionByProtocolChannel(UUID channelID) {
         if (subIterator.hasNext()) {
             return subIterator.next();
         } else {
-            log.warn("No subscription found for channel ID " + channelID);
+            if (log.isDebugEnabled()) {
+                log.debug("No subscription found for channel ID " + channelID);
```

Comment:
We get a channel flow request to a channel for which a subscription is not yet registered. How can we handle that?